app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Api::V1::ReservationsController < ApplicationController
+  before_action :set_reservation, only: :destroy
+
+  def index
+    @reservations = current_user.reservations
+  end
+
+  def show
+    render json: @reservation, status: :ok
+  end
```

Comment:
- [ ] I believe you are not using this action, maybe it's better to remove it.

Reply:
Yes, you're right. It has been removed!